ext/zend_test/tests/get_function_or_method_name_01.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+get_function_or_method_name when included file is scoped
+--EXTENSIONS--
+zend_test
```

Comment:
This is unlikely to work for the PHP-8.0 branch (only properly supported for PHP-8.1+). For PHP-8.0, we need to skip the test, if the extension is not loaded:
```suggestion
--SKIPIF--
<?php
if (!extension_loaded("zend_test")) die("skip zend_test extension not available");
?>
```

Reply:
done. And maybe [observer_bug81430_1.phpt](https://github.com/php/php-src/blob/PHP-8.0/ext/zend_test/tests/observer_bug81430_1.phpt) and [observer_bug81430_2.phpt](https://github.com/php/php-src/blob/PHP-8.0/ext/zend_test/tests/observer_bug81430_2.phpt) should be fixed then?

Reply:
Nice catch! Yes, they should be fixed as well

Reply:
[Done](https://github.com/php/php-src/commit/710294c9a56cd69364d699a840e24f2600021199).